.github/workflows/deploy_production.yml

```diff
@@ -7,6 +7,14 @@ on:
   workflow_dispatch:
 
 jobs:
+  build_and_push_image:
+      name: Build and Push Image
+      uses: zooniverse/ci-cd/.github/workflows/build_and_push_image.yaml@main
+      with:
+        repo_name: panoptes
+        commit_id: ${{ github.sha }}
+        latest: true
```

Comment:
Don't set the latest on production builds - this should be reserved for master branch builds. Instead rely on the GH sha id.
```suggestion
```